changelogs/fragments/8826-keycloak-component.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - keycloak_component - add plugin for managing any keycloak component (https://github.com/ansible-collections/community.general/pull/8826)
```

Please use punctuation here, this is later exported to public documentation.
```suggestion
  - keycloak_component - add plugin for managing any keycloak component (https://github.com/ansible-collections/community.general/pull/8826).
```

---

This file needs to be removed anyway.